nml/version_info.py

```diff
@@ -139,11 +139,11 @@ def get_lib_versions():
     #PIL
     try:
         from PIL import Image
-        versions["PIL"] = Image.PILLOW_VERSION
+        versions["PIL"] = Image.__version__
     except ImportError:
         try:
             import Image
-            versions["PIL"] = Image.PILLOW_VERSION
+            versions["PIL"] = Image.__version__
```

Comment:
Also, maybe, while here, try to remove this code duplication by restructuring something?

Reply:
TBH, I have no clue, how to write this better, since it checks for 2 different imports. Sorry, I am not a native python coder.